lib3h/src/p2p_protocol.rs

```diff
@@ -0,0 +1,10 @@
+/// Enum holding all message types in the 'network module <-> network module' protocol.
+/// TODO
+#[derive(Debug, Clone, PartialEq)]
+pub enum P2pProtocol {
```

Comment:
let's not include this yet, we're not ready to tackle p2p messaging

Reply:
ok